chainlit_app/services/chat_service.py

```diff
@@ -0,0 +1,29 @@
+import os
+
+import requests
+
+API_URL = os.getenv("API_URL", "http://localhost:8000")
+API_SECRET_KEY = os.getenv("SECRET_KEY", "my_secret_key")  # Default if not set in env
+
+
+def get_chat_response(user_message: str) -> dict:
+    """
+    Sends the user's message to the chat API and retrieves the response.
+
+    Args:
```

Comment:
We have been using [`numpy` style](https://numpydoc.readthedocs.io/en/latest/format.html) docstrings so far. Suggest changing args to `Parameters` and underlining